app/src/main/java/cm/aptoide/pt/app/view/MoreBundlePresenter.java

```diff
@@ -111,13 +115,20 @@ private void handleError(HomeBundlesModel.Error error) {
     view.getLifecycleEvent()
         .filter(lifecycleEvent -> lifecycleEvent.equals(View.LifecycleEvent.CREATE))
         .flatMap(created -> view.appClicked()
-            .doOnNext(click -> homeAnalytics.sendTapOnAppInteractEvent(click.getApp()
-                    .getRating(), click.getApp()
-                    .getPackageName(), click.getAppPosition(), click.getBundlePosition(),
-                click.getBundle()
-                    .getTag(), click.getBundle()
-                    .getContent()
-                    .size()))
+            .doOnNext(click -> {
+              homeAnalytics.sendTapOnAppInteractEvent(click.getApp()
+                      .getRating(), click.getApp()
+                      .getPackageName(), click.getAppPosition(), click.getBundlePosition(),
+                  click.getBundle()
+                      .getTag(), click.getBundle()
+                      .getContent()
+                      .size(), chipManager.getCurrentChip());
+              if (chipManager.getCurrentChip() != null) {
+                homeAnalytics.sendChipTapOnApp(click.getBundle()
+                    .getTag(), click.getApp()
+                    .getPackageName(), chipManager.getCurrentChip());
```

Comment:
I would just pass the Name of the chip instead of the whole chip, which is what we need... On this one we wouldn't have the problem that we have above, since we are already verifying if the chip is not null